src/components/utils/__mocks__/objectUrl.ts

```diff
@@ -0,0 +1,2 @@
+export const createObjectURL = jest.fn().mockReturnValue('fake-object-url')
+export const revokeObjectURL = jest.fn()
```

Comment:
I'm not 100% sure, but it looks like this file and `src/components/utils/objectUrl/__mocks__/index.ts` further down below are duplicates & 1 of them is unused.

Reply:
Yes it was [a known issue](https://github.com/facebook/jest/issues/2070)